src/utils/query_ops.js

```diff
@@ -0,0 +1,130 @@
+
+import lodashGroupBy from 'lodash/groupBy';
+import map from 'lodash/map';
+import flatten from 'lodash/flatten';
+import lodashFilter from 'lodash/filter';
+import toPairs from 'lodash/toPairs';
+import chunk from 'lodash/chunk';
+import unzip from 'lodash/unzip';
+import sortBy from 'lodash/sortBy';
+import fromPairs from 'lodash/fromPairs';
+
+class Wrapper {
+  constructor(list) {
+    this.list = list;
+  }
+
+  toArray() {
+    return this.list;
+  }
+}
+
+
+// Add a chainable method to Wrapper
+const extend_wrapper = (methods) => {
+  toPairs(methods).forEach(([name, method]) => {
+    // USE function(){} DECLARATION TO BIND this AT CALL TIME
+    Wrapper.prototype[name] = function anonymous(...args) {
+      this.list = method(this.list, ...args);
+      return this;
+    };
+  });
+};
+
+
+const frum = (list) => {
```

Comment:
Yes, I like the word `from` because it makes the chained functions read like SQL, but it is reserved. We can go back to `chain`, but that word alludes to the chaining of functions, which is about code. while the other functions (map, filter, groupby) are words that describe action on data.

Other words could be `using` or `with`.  If `frum` is a default export, then it is private, and then does the name even matter? The importer gets to decide the name:

    import magicalChainFunction from queryOps